lib/seed_migration/engine.rb

```diff
@@ -4,6 +4,14 @@ module SeedMigration
 
   DEFAULT_TABLE_NAME = 'seed_migration_data_migrations'
 
+  # Default environments list set, defaults to []
+  mattr_accessor :environments_defaults
+  # set the current enviroment rails value, defaults to Rails.env
+  mattr_accessor :environments_rails_value
+  # set if the environments method should explicitly be called in the seeds, defaults to false
+  mattr_accessor :environments_required
+  # set the seed version from where the environments check should start, defaults to nil (which will check it from the first seed file)
+  mattr_accessor :environments_version_bootstrap
```

Comment:
@cairesr Just for my understanding, if this isn't `nil`, what might it be set to and why?

Reply:
We might set it to the first migration that should start validating the environment through this new change. If we don't do this, then we'd need to add environments to all previous migrations so far. And an eventual rollback on PROD might fail for previous migrations, since they will not have environments set.

So say that we add a seed migration after turning it on on Swoop, and it's called `20200831_add_new_column_to_job.rb`. We set this as the start point to check for envs, so from this point on, all new seeds will require it. Previous ones will stay untouched, again, for rollback consistency.

Let me know if it clarifies it.

Reply:
Great, thanks for the clarification!